### PR TITLE
 Horreum report contains mixes CPU limit and request #701

### PR DESCRIPTION
**Pull request by andyuk1986**

### CODE REVIEW

File: provision/common/Taskfile.yaml [hunk 1]
```diff
@@ -93,7 +95,8 @@ tasks:
           --arg kc_repo "{{ .KC_REPOSITORY }}" \
           --arg kc_branch "{{ .KC_BRANCH }}" \
           --argjson is_multi_az "{{ .MULTI_AZ }}" \
-          '. + { "numOfPods": ($num_of_pods|tonumber), "cpuLimitPerPod": ($cpu_limit|tonumber),
+          '. + { "numOfPods": ($num_of_pods|tonumber), "cpuRequestsPerPod": ($cpu_requests_per_pod|tonumber),
+                             "cpuLimitsPerPod": (if $cpu_limits_per_pod | length == 0 then null else ($cpu_limits_per_pod | tonumber?) end),
```
Comment (kami619) [inline provision/common/Taskfile.yaml:99]: can this be re-written as `(if ($cpu_limits_per_pod | length) == 0 then null else ($cpu_limits_per_pod | tonumber?) end)` @andyuk1986 ?

**Comment by kami619** [comment]
apart from the small nitpic above, the change looks good to me @andyuk1986

**Review by kami619 — APPROVED** [review]
need a little clarification on the jq if, else condition, apart from that the change looks good.

**Comment by andyuk1986** [comment]
@kami619 hey Kamesh, thanks for the merge. I was on PTO yesterday, and today half day, just have seen your message. Thanks for updating the PR. Regards,Anna.

**Pull request merged**